src/main/java/org/spongepowered/common/service/permission/SpongePermissionService.java

```diff
@@ -180,4 +182,11 @@ public void addDescription(PermissionDescription permissionDescription) {
     public SpongeSubjectCollection getDefaultCollection() {
         return defaultCollection;
     }
+
+    public void setupDefaultPermissions() {
+        final SubjectData opGroupLevel1Data = getGroupForOpLevel(1).getSubjectData();
+        opGroupLevel1Data.setPermission(SubjectData.GLOBAL_CONTEXT, "minecraft.selector", Tristate.TRUE);
```

Comment:
`CommandPermissions.SELECTOR_PERMISSION` and `CommandPermissions.SELECTOR_LEVEL`